### PR TITLE
Update SDK and JSON.NET package references

### PR DESCRIPTION
**Pull request by EasyRhinoMSFT**

### CODE REVIEW

File: src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj [hunk 1]
```diff
@@ -291,20 +291,19 @@
       <HintPath>..\packages\Microsoft.VisualStudio.Validation.15.3.58\lib\net45\Microsoft.VisualStudio.Validation.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
```
Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj]: >true [](start = 26, length = 4)

Yay!

File: src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj [hunk 2]
```diff
@@ -463,19 +462,4 @@
...
-  back into the list of items included in the VSIX.
-
-  See: https://social.msdn.microsoft.com/Forums/en-US/4196345b-036d-479e-a6b5-b2b20f266329/vsix-newtonsoft-isnt-in-package-vs155-suppress-package?forum=vsx
-  -->
-  <Target Name="ValidateVsixParts" DependsOnTargets="$(ValidateVsixPartsDependsOn)">
-    <ItemGroup>
-      <VSIXSourceItem Include="..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll" />
-      <VSIXSourceItem Include="..\packages\System.Collections.Immutable.1.5.0\lib\netstandard2.0\System.Collections.Immutable.dll" />
-    </ItemGroup>
-  </Target>
```
Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj]: >Target [](start = 4, length = 6)

Double yay!

File: src/Sarif.Viewer.VisualStudio/Sarif/Invocation.extensions.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@ public static InvocationModel ToInvocationModel(this Invocation invocation)
                 Account = invocation.Account,
                 ProcessId = invocation.ProcessId,
                 FileName = invocation.ExecutableLocation?.Uri?.ToString(),
-                WorkingDirectory = invocation.WorkingDirectory,
+                WorkingDirectory = invocation.WorkingDirectory?.Uri?.ToString(),
```
Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio/Sarif/Invocation.extensions.cs]: >ToString [](start = 69, length = 8)

Perhaps `.OriginalString`?

File: src/Sarif.Viewer.VisualStudio.Interop/Sarif.Viewer.VisualStudio.Interop.csproj [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup Label="AssemblyAttributes">
     <AssemblyTitle>SARIF Viewer Interop Library</AssemblyTitle>
-    <Description>Provides an interop API that enables Visual Studio extensions to consume services provided by the SARIF Viewer extension.</Description>
+    <Description>Provides a simple API that allows Visual Studio extensions to easily consume services exposed by the Microsoft SARIF Viewer extension.</Description>
```
Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio.Interop/Sarif.Viewer.VisualStudio.Interop.csproj]: >simple [](start = 28, length = 6)

"simple", "easily"... Well, I suppose there's nothing wrong with a bit of marketing. If it's me, I omit both those words. This is a public-facing string, so perhaps @michaelcfanning has an opinion.

**Comment by lgolding** [comment]
One wordsmithing nit, one possible functional issue. Otherwise LGTM.

File: src/Sarif.Viewer.VisualStudio/Sarif/Invocation.extensions.cs (hunk 3 above)
Comment (EasyRhin0) [reply src/Sarif.Viewer.VisualStudio/Sarif/Invocation.extensions.cs]: I was following what FileName does. Not really important, as this property isn't currently used.

---
In reply to: [217543302](https://github.com/Microsoft/sarif-visualstudio-extension/pull/34#discussion_r217543302) [](ancestors = 217543302)

File: src/Sarif.Viewer.VisualStudio.Interop/Sarif.Viewer.VisualStudio.Interop.csproj (hunk 4 above)
Comment (EasyRhin0) [reply src/Sarif.Viewer.VisualStudio.Interop/Sarif.Viewer.VisualStudio.Interop.csproj]: Fair enough, will dry it out :)

---
In reply to: [217543599](https://github.com/Microsoft/sarif-visualstudio-extension/pull/34#discussion_r217543599) [](ancestors = 217543599)

**Pull request merged**